lib/src/widgets/yust_select_form.dart

```diff
@@ -76,6 +76,7 @@ class YustSelectForm<T> extends StatelessWidget {
                     const EdgeInsets.symmetric(horizontal: 24, vertical: 0),
                 child: TextFormField(
                   initialValue: searchValue,
+                  autofocus: true,
```

Comment:
Make optional - this widget is not necessarily in a dialog, therefore it should not always have the autofocus.